### DISCUSSION TITLE
Feature Request: Controlling zoom by JavaScript

### DISCUSSION THREAD

**Discussion by fbenedetti06** [google/model-viewer#3375] [Q&A]
Is there a way to control Zoom/Camera/Orbit by program?
No "set" function in [Staging & Cameras][Methods]!

I will have to disable zoom because users zoom the model each time they want to scroll the web page with mouse wheel.
I know I'm not the only one with this trouble.

My idea is to be able to create a slider on side of model to keep the ability to zoom...

**Comment by elalish** [comment]
I would recommend https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-disable-zoom and https://modelviewer.dev/docs/#entrydocs-stagingandcameras-methods-zoom and https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-cameraOrbit

All attributes are "set" methods: just like normal HTML, any hyphenated-attribute can be set in JS via camelCase: `mv.cameraOrbit = "auto auto auto";`.
